app/views/admin/claims/_answer_section.html.erb

```diff
@@ -0,0 +1,19 @@
+<% unless heading.nil? %>
```

Comment:
Every call to this partial includes a heading so this conditional is not necessary.